packages/dart_frog/lib/src/serve.dart

```diff
@@ -16,6 +16,7 @@ Future<HttpServer> serve(
   int port, {
   String? poweredByHeader = 'Dart with package:dart_frog',
   SecurityContext? securityContext,
+  bool shared = false,
```

Comment:
Can we add some documentation to this parameter in the functions Dart doc?

I was thinking something like:

```dart
/// ...
/// If a [securityContext] is provided an HTTPS server will be started.
///
/// Enable [shared] to allow additional [HttpServer] objects to bind to the
/// same combination of [HttpServer.address] and [HttpServer.port].
```

Reply:
Thank you, this would be my first time doing something like this so I'm missing the proper etiquette